socketIO_client/__init__.py

```diff
@@ -203,6 +207,8 @@ def _close(self):
             self._transport_instance.send_packet(engineIO_packet_type)
         except (TimeoutError, ConnectionError):
             pass
+        finally:
+            self._transport_instance.disconnect()
```

Comment:
Don't know, the send_packet(engineIO_packet_type) sends disconnect packet so might be that we want to do that so then it wouldn't matter with the heartbeat thread exception since it might be that heartbeat thread has been closed but transport is still open